pkg/Microsoft.NETFramework.Compatibility.Shims/Microsoft.NETFramework.Compatibility.Shims.pkgproj

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <PackageVersion>$(CompatibilityShimsPackageVersion)</PackageVersion>
+    <_ShimsLocationPath>$(BaseIntermediateOutputPath)shims/netcoreapp/facades/</_ShimsLocationPath>
+    <_PkgLibDir>/lib/netcoreapp2.0/</_PkgLibDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ShimsToInclude Include="mscorlib.dll" />
+    <ShimsToInclude Include="System.dll" />
+    <ShimsToInclude Include="System.Configuration.dll" />
+    <ShimsToInclude Include="System.Core.dll" />
+    <ShimsToInclude Include="System.Data.dll" />
+    <ShimsToInclude Include="System.Drawing.dll" />
+    <ShimsToInclude Include="System.Net.dll" />
+    <ShimsToInclude Include="System.Security.dll" />
+    <ShimsToInclude Include="System.ServiceModel.Web.dll" />
+    <ShimsToInclude Include="System.ServiceProcess.dll" />
+    <ShimsToInclude Include="System.Transactions.dll" />
+    <ShimsToInclude Include="WindowsBase.dll" />
+  </ItemGroup>
+
+  <Target Name="AddShimsToPackage"
+          BeforeTargets="GetPackageFiles">
+
+    <Error Text="Required facade: %(ShimsToInclude.Identity) was not found in $(_ShimsLocationPath)." Condition="!Exists('@(ShimsToInclude->'$(_ShimsLocationPath)%(Identity)')')" />
+
+    <ItemGroup>
+      <File Include="@(ShimsToInclude->'$(_ShimsLocationPath)%(Identity)')">
+        <TargetPath>$(_PkgLibDir)%(ShimsToInclude.Identity)</TargetPath>
```

Comment:
NIT: Do you actually need `%(ShimsToInclude.Identity)` here? I thought that you could just do the lib dir and the file name would just remain the same.

Reply:
I think you're right. I'm testing this locally and if it works I will update the PR.

Reply:
Fixed.